ngraph/frontend/onnx_import/src/utils/onnx_test_util.hpp

```diff
@@ -0,0 +1,54 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include <string>
+
+#include "onnx_import/utils/onnx_importer_visibility.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        struct ONNX_IMPORTER_API ComparisonResult
+        {
+            ComparisonResult() = default;
+            ComparisonResult(std::string error)
+                : is_ok{false}
+                , error_message{std::move(error)}
+            {
+            }
+            ComparisonResult(ComparisonResult&&) = default;
+            ComparisonResult(const ComparisonResult&) = default;
+            ComparisonResult& operator=(ComparisonResult&&) = default;
+            ComparisonResult& operator=(const ComparisonResult&) = default;
+
+            bool is_ok = true;
+            std::string error_message;
+
+            static ComparisonResult pass() { return {}; }
+            static ComparisonResult fail(std::string error)
+            {
+                return ComparisonResult{std::move(error)};
+            }
+        };
+
+        ONNX_IMPORTER_API ComparisonResult
+            compare_onnx_models(const std::string& model, const std::string& reference_model_path);
```

Comment:
Why do we need to have these struct and method inside the ONNX importer if we use their only for test purposes?

Reply:
This was moved here to avoid protobuf related problems. When the editor is extracted to a separate library this can perhaps be moved somewhere else.

Reply:
@tomdol Can we try to avoid such code? Maybe we can make a call and discuss it, because for me looks strange that classes and methods which are used only in the tests, we need to export and have in the core library.

Reply:
Sure, we can sync up about it. But just FYI @mbencer is working on the extraction of everything editor-related to a separate library so he will move this code too.

Reply:
As discussed, I've temporarily switched to comparison of binary strings containing the extracted models. I've also created a ticket to address it after the editor becomes a separate library: 49339